tap_linkedin_ads/schemas/campaigns.json

```diff
@@ -217,6 +217,30 @@
         "string"
       ]
     },
+    "total_budget":{
+      "type": "object",
```

Comment:
Can you please add null in the data type with an object?

Reply:
Added null type along with object.